app/src/main/java/org/sopt/and/MyActivity.kt

```diff
@@ -0,0 +1,194 @@
+package org.sopt.and
+
+import android.os.Bundle
+import android.widget.Space
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.enableEdgeToEdge
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.sopt.and.ui.theme.ANDANDROIDTheme
+
+class MyActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        enableEdgeToEdge()
+        setContent {
+            ANDANDROIDTheme {
+                val email = intent.getStringExtra("email")
+                if (email != null) {
+                    myView(email)
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun headerProfile(email: String) {
```

Comment:
```suggestion
fun HeaderProfile(email: String) {
```
UI를 그리는 Composable 함수의 경우 대문자 시작이 컨벤션입니다.